src/main/java/com/spikes2212/dashboard/Namespace.java

```diff
@@ -20,8 +20,12 @@ public interface Namespace {
 
     void putData(String key, Sendable value);
```

Comment:
maybe add a `putCommand`method, which gets a `Command` and sends it to `putData` with an `ignoringDisable`